### ISSUE TITLE
Presets Panel: Icon Buttons Missing Accessible Name 

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#5453]
## Issue Description
The icon buttons in the presets menu do not have accessible names, so there is no programmatic way to identify each button or distinguish one from the other.

Related to: 

- #5450 
- #5451 
- #5452
- #5489

Derived from #5187 .

## WCAG Criteria
[3.3.2 Labels or Instructions - A](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)

## Notes
Use `aria-label` to give the buttons a unique, descriptive accessible name. For example, `aria-label="Delete Preset"`. Also, hide the SVG image to avoid unnecessary or redundant announcements with the `aria-hidden="true"` attribute.

## Screenshot or Screen Recording

![Image](https://github.com/user-attachments/assets/ea5613c7-29e0-46bf-b887-ce4ad0c09214)

## Relevant Code
```html
<button class="m-0 h-full rounded-md bg-transparent p-2 text-gray-400 hover:text-gray-700 dark:text-gray-400 dark:hover:text-gray-200">
```
